v2/stack_analyses.yaml

```diff
@@ -0,0 +1,455 @@
+openapi: 3.0.1
+info:
+  title: Stack Analyses API
+  version: "2.0"
+servers:
+  - url: https://f8a-analytics-2445582058137.{environment}.gw.apicast.io/api/v2
+    variables:
+      environment:
+        default: api    # Production server
+        enum:
+          - production # Production server
+          - staging # Staging server
+security:
```

Comment:
What does this field referring to ? Please add documentation.

Reply:
This field refers Authorization through 3scale `user_key`. There is no way to add description field for this, I added a comment for more clarity.